department-of-veterans-affairs/va.gov-cms
# VACMS-6455: add audit tool for VAMC system service headings

## Description

Closes #6455 

## Testing done

Visual Testing

## Screenshots

<img width="1775" alt="Screen Shot 2021-10-20 at 11 24 00 AM" src="https://user-images.githubusercontent.com/21045418/138123051-a8e03178-e814-4369-a9b1-aabd82e62ed3.png">

Filters applied

<img width="1742" alt="Screen Shot 2021-10-20 at 11 24 17 AM" src="https://user-images.githubusercontent.com/21045418/138123097-53218752-124b-4dba-ad1e-7f5d9b173779.png">

CSV button is present

![Screen Shot 2021-10-20 at 11 24 57 AM](https://user-images.githubusercontent.com/21045418/138123157-07e5a405-e19c-4df1-98f9-37696a0ddebe.png)

Exported CSV file contains rows matching the applied filters

## QA steps

As user with either the administrator role or the content admin role

1. [x] Visit /admin/content/audit/vamc-system-service-audit and verify the page loads
2. [x] Filter the view by VAMC System Description of Service **contains** `<h3>Care we provide at</h3>`
3. [x] Verify the view refreshes and each item includes this heading
4. [x] Scroll to the bottom and click the CSV icon to download a CSV of your filtered results
5. [x] Verify that the CSV contains the same set of filtered items
6. [x] Reset the view filters and filter the view by VAMC System Description of Service **does not contain** `<h3>`
7. [x] Verify the view refreshes and none of the items contain an h3 tag
8. [x] Scroll to the bottom and click the CSV icon to download a CSV of your filtered results
9. [x] Verify the CSV contains the same set of filtered items

As a user without the administrator or content admin roles

1. [x] Visit /admin/content/audit/vamc-system-service-audit and verify you do not have access
